ui-modules/home/app/views/about/about.template.html

```diff
@@ -77,8 +77,6 @@ <h2>HA status</h2>
                 </tr>
                 </tbody>
             </table>
-            <h2>Logbook</h2>
-            <a class="btn btn-sm btn-default" href="#!/logbook"><i class="fa fa-eye"></i> Logbook</a>
```

Comment:
Logbook was moved down as a directive, not as a state. Button looks like not required.

Reply:
Open to suggestions on how to hide/display the logbook, e.g. return back the button, accordion, etc.